aliasgen.py

```diff
@@ -46,18 +47,18 @@ def scan_dir(folder):
 
                 alias_file += module_all_out
 
-
             with (alias_folder / unprefixed).open("w+") as f:
                 f.write(alias_file)
             sleep(0.01)
             # Test that it works
             import_name = "discord" + import_name[len("nextcord"):]
             try:
                 import_module(import_name)
-            except:
+            except ModuleNotFoundError:
```

Comment:
We are checking for broken shit like typeerrors here too!

Reply:
maybe checking for Exception instead of using a bare except? or do you want to catch keyboard interrupts aswell?

Reply:
I don't see where a keyboard interrupt would happen here

Reply:
Scenario: There is a problem siomewhere resulting in the code to hang up there for whatever reason. In order of stopping you would normally use a keyboard interrupt. instead of stopping you would have to repeat this for every single submodule now to stop the entire code. See PEP8 guide for more infos

>When catching exceptions, mention specific exceptions whenever possible instead of using a bare except: clause.
>[...]
> A bare except: clause will catch SystemExit and KeyboardInterrupt exceptions, making it harder to interrupt a program with Control-C, and can disguise other problems. If you want to catch all exceptions that signal program errors, use except Exception: (bare except is equivalent to except BaseException:).
> A good rule of thumb is to limit use of bare ‘except’ clauses to two cases:
> 
> If the exception handler will be printing out or logging the traceback; at least the user will be aware that an error has occurred.
> If the code needs to do some cleanup work, but then lets the exception propagate upwards with raise. try...finally can be a better way to handle this case.

Reply:
Hmmm fair. Maybe change to anything but keyboard interrupt

Reply:
I changed it to Exception